test/SdkTests/TestConfig.xml

```diff
@@ -37,7 +37,6 @@
     <Method Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToPublishASelfContainedApp.It_can_make_a_Windows_GUI_exe" />
     <Class Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToPublishASingleFileApp" />
     <Class Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToPublishAWebApp" />
-    <Class Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToRunCrossgen" />
```

Comment:
This test doesn't exist anymore. I found it existing in 2019 but I didn't walk the entire file history to see when it was deleted.

Reply:
This sort of thing will hopefully become easier once everything is one repo

Reply:
Well, this file isn't being migrated since it was for running SDK tests in this repo. I went through and confirmed these tests being in the SDK repo already, so I don't need to do anything with the entire SdkTests project.